index.js

```diff
@@ -1,39 +1,44 @@
 (function(root) {
+  var docEl = document && document.documentElement;
   var scrollbarSize;
+  var scrollTop;
 
   function getScrollbarSize() {
     if (typeof scrollbarSize !== 'undefined') return scrollbarSize;
 
-    var doc = document.documentElement;
```

Comment:
Isn't caching it above ok though? Since it will still come back as false if it was rendered server side. Then we don't have to grab it every time.